### DISCUSSION TITLE
Add Vue template?

### DISCUSSION THREAD

**Discussion by jcbhmr** [devcontainers-community/templates#17] [Ideas]
im looking at the 2nd most popular template on vscode-dev-containers: vue. its clearly still very popular

https://github.com/orgs/microsoft/packages?sort_by=downloads_desc&q=vscode-dev-containers

![image](https://user-images.githubusercontent.com/61068799/232183769-c43f1296-2cd5-4fac-80da-21a5bd4ea357.png)

I don't know if it's complicated enough to warrant a full template though? could it be folded into a js-based template instead?

https://github.com/microsoft/vscode-dev-containers/tree/main/containers/vue

then again, it does have these that are _vue specific_ addins:
- vscode tasks.json -- do devcontainer templates support _other files_ besides `.devcontainer/*`? 🤔
- npm install -g http-server @vue/cli @vue/cli-service-global
- "octref.vetur" vscode extension
other than that not much...

thoughts? looking for opinions on if this is another template like #2 that we want to add to this @devcontainers-community project or not

**Comment by eitsupi** [comment]
Thanks for taking this up.
I was wondering about this as well.
I don't know if it is worth daring to have a separate template since it is almost same to the default js template.

**Reply by jcbhmr** [reply]
i think you and i are of the same mind: to similar to default js template
only significant difference is the vetur vscode extension https://marketplace.visualstudio.com/items?itemName=octref.vetur

sidenote: i would like to see a note or demo like `npm install -g your-global-package-here` in https://github.com/devcontainers/templates/blob/main/src/javascript-node/.devcontainer/devcontainer.json#L14 but that's just me
maybe i should open a pr 😜
